src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -92,6 +92,10 @@ public final void run() {
         if (!Bukkit.isPrimaryThread()) {
             return;
         }
+        
+        if (!SlimefunPlugin.isEnabled()) {
```

Comment:
```suggestion
        if (!SlimefunPlugin.instance().isEnabled()) {
```

Reply:
instance() is nullable. So a static isEnabled() method might be a good idea, check if it isnt null and then do the normal isEnabled call.

Reply:
instance will never be null, because before we set it to null, we cancel tasks anyway.

Reply:
Cancelling will only cancel the scheduling of new tasks, it won't terminate tasks which have already been started. And since this is running asynchronously, it may run just in time when instance is null and tasks have already been cancelled.

Reply:
> Cancelling will only cancel the scheduling of new tasks, it won't terminate tasks which have already been started. And since this is running asynchronously, it may run just in time when instance is null and tasks have already been cancelled.

This isn't run asynchronously.